src/api/spec/controllers/webui/packages/job_history_controller_spec.rb

```diff
@@ -11,28 +11,23 @@
       repo
     end
 
+    # FIXME: The before filter this is testing (set_repository) is in Webui::WebuiController
```

Comment:
Is it possible to simply move the tests?  If not, the FIXME can probably disappear; it's not hard to find where things live.

Reply:
Don't want to mess with the anoynmous rspec controller we use to test `Webui::WebuiController`, it's out of actions to tests behavior of filters. That needs cleanup...